{
  "results": [
    {
      "columns": [],
      "data": []
    }
  ],
  "errors": [
    {
      "code": "Neo.DatabaseError.Transaction.CouldNotCommit",
      "message": "javax.transaction.HeuristicRollbackException: Failed to commit transaction Transaction(17, owner:\"qtp797381996-56\")[STATUS_NO_TRANSACTION,Resources=1], transaction rolled back ---> javax.transaction.xa.XAException",
      "stackTrace": "java.lang.RuntimeException: javax.transaction.HeuristicRollbackException: Failed to commit transaction Transaction(17, owner:\"qtp797381996-56\")[STATUS_NO_TRANSACTION,Resources=1], transaction rolled back ---> javax.transaction.xa.XAException\n\tat org.neo4j.server.rest.transactional.TransitionalTxManagementKernelTransaction.commit(TransitionalTxManagementKernelTransaction.java:88)\n\tat org.neo4j.server.rest.transactional.TransactionHandle.commit(TransactionHandle.java:195)\n\tat org.neo4j.server.rest.transactional.TransactionHandle.commit(TransactionHandle.java:109)\n\tat org.neo4j.server.rest.web.TransactionalService$2.write(TransactionalService.java:198)\n\tat com.sun.jersey.core.impl.provider.entity.StreamingOutputProvider.writeTo(StreamingOutputProvider.java:71)\n\tat com.sun.jersey.core.impl.provider.entity.StreamingOutputProvider.writeTo(StreamingOutputProvider.java:57)\n\tat com.sun.jersey.spi.container.ContainerResponse.write(ContainerResponse.java:306)\n\tat com.sun.jersey.server.impl.application.WebApplicationImpl._handleRequest(WebApplicationImpl.java:1437)\n\tat com.sun.jersey.server.impl.application.WebApplicationImpl.handleRequest(WebApplicationImpl.java:1349)\n\tat com.sun.jersey.server.impl.application.WebApplicationImpl.handleRequest(WebApplicationImpl.java:1339)\n\tat com.sun.jersey.spi.container.servlet.WebComponent.service(WebComponent.java:416)\n\tat com.sun.jersey.spi.container.servlet.ServletContainer.service(ServletContainer.java:537)\n\tat com.sun.jersey.spi.container.servlet.ServletContainer.service(ServletContainer.java:699)\n\tat javax.servlet.http.HttpServlet.service(HttpServlet.java:848)\n\tat org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:698)\n\tat org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1506)\n\tat org.neo4j.server.rest.security.SecurityFilter.doFilter(SecurityFilter.java:112)\n\tat org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1477)\n\tat org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:503)\n\tat org.eclipse.jetty.server.session.SessionHandler.doHandle(SessionHandler.java:211)\n\tat org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1096)\n\tat org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:432)\n\tat org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:175)\n\tat org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1030)\n\tat org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:136)\n\tat org.eclipse.jetty.server.handler.HandlerList.handle(HandlerList.java:52)\n\tat org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:97)\n\tat org.eclipse.jetty.server.Server.handle(Server.java:445)\n\tat org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:268)\n\tat org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:229)\n\tat org.eclipse.jetty.io.AbstractConnection$ReadCallback.run(AbstractConnection.java:358)\n\tat org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:601)\n\tat org.eclipse.jetty.util.thread.QueuedThreadPool$3.run(QueuedThreadPool.java:532)\n\tat java.lang.Thread.run(Thread.java:744)\nCaused by: javax.transaction.HeuristicRollbackException: Failed to commit transaction Transaction(17, owner:\"qtp797381996-56\")[STATUS_NO_TRANSACTION,Resources=1], transaction rolled back ---> javax.transaction.xa.XAException\n\tat org.neo4j.kernel.impl.transaction.TxManager.commit(TxManager.java:569)\n\tat org.neo4j.kernel.impl.transaction.TxManager.commit(TxManager.java:403)\n\tat org.neo4j.server.rest.transactional.TransitionalTxManagementKernelTransaction.commit(TransitionalTxManagementKernelTransaction.java:84)\n\t... 33 more\nCaused by: javax.transaction.xa.XAException\n\tat org.neo4j.kernel.impl.nioneo.xa.IntegrityValidator.validateNodeRecord(IntegrityValidator.java:54)\n\tat org.neo4j.kernel.impl.nioneo.xa.NeoStoreTransaction.doPrepare(NeoStoreTransaction.java:402)\n\tat org.neo4j.kernel.impl.transaction.xaframework.XaTransaction.prepare(XaTransaction.java:307)\n\tat org.neo4j.kernel.impl.transaction.xaframework.XaResourceManager.commitWriteTx(XaResourceManager.java:539)\n\tat org.neo4j.kernel.impl.transaction.xaframework.XaResourceManager.commit(XaResourceManager.java:490)\n\tat org.neo4j.kernel.impl.transaction.xaframework.XaResourceHelpImpl.commit(XaResourceHelpImpl.java:64)\n\tat org.neo4j.kernel.impl.transaction.TransactionImpl.doCommit(TransactionImpl.java:544)\n\tat org.neo4j.kernel.impl.transaction.TxManager.commit(TxManager.java:464)\n\t... 35 more\nCaused by: org.neo4j.graphdb.ConstraintViolationException: Node record Node[11,used=false,rel=29,prop=18,labels=Inline(0x0:[]),light] still has relationships\n\t... 43 more\n"
    }
  ]
}
